src/cargo/util/context/mod.rs

```diff
@@ -91,6 +91,7 @@ use serde::Deserialize;
 use serde_untagged::UntaggedEnumVisitor;
 use time::OffsetDateTime;
 use toml_edit::Item;
+use tracing::warn;
```

Comment:
This looks like dead code?

Reply:
Resolved in 5afdc115d295f74c475a2252df40ff7d2351c3eb